frontend/src/components/assessments/TestSubmissionMessage.tsx

```diff
@@ -1,17 +1,23 @@
-import React from "react";
+import React, { useContext } from "react";
 import { Center } from "@chakra-ui/react";
 
 import TestSubmissionIllustration from "../../assets/illustrations/test-submission.svg";
-import { HOME_PAGE } from "../../constants/Routes";
+import AuthContext from "../../contexts/AuthContext";
 import MessageContainer from "../common/MessageContainer";
 
 const TestSubmissionMessage = (): React.ReactElement => {
+  const { setAuthenticatedUser } = useContext(AuthContext);
+
+  const handleClick = () => {
+    setAuthenticatedUser(null);
+  };
+
   return (
     <Center minHeight="100vh">
       <MessageContainer
-        buttonRoute={HOME_PAGE}
         buttonText="Go to Home Page"
         image={TestSubmissionIllustration}
+        onClick={handleClick}
         paragraphs={[]}
         subtitle="Great Job! Your test submission was a success. 
```

Comment:
Just noticed this, it's really a design thing, but given the demographic that will be reading this copy, I'm wondering if we could simplify the wording? Something like `Great job! We've saved your answers.` might be a little more understandable maybe?

Reply:
i like!